config/jobs/kubernetes-sigs/ingress-controller-conformance/ingress-controller-conformance.yaml

```diff
@@ -59,7 +59,7 @@ presubmits:
         args:
         - bash
         - -c
-        - apt update && apt-get install -y python3-pip && pip3 install reformat-gherkin && make verify-gherkin
+        - pip3 install reformat-gherkin==2.0.0 --ignore-installed PyYAML && make verify-gherkin
```

Comment:
we really shouldn't need a workaround like this in a job, the image changes are clearly broken.